1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,44 @@
 // Write your code here
+const items = document.querySelectorAll('img');
+const nextItem = document.querySelector('.next');
+const previousItem = document.querySelector('.previous');
+
+let count = 0;
+
+function showNextItem() {
+  items[count].classList.remove('active');
+  if(count < items.length - 1) {
+    count++;
+  } else {
+    count = 0;
+  }
+  items[count].classList.add('active');
+  }
+
+function showPreviousItem() {
+  items[count].classList.remove('active');
+  if(count > 0) {
+    count--;
+  } else {
+    count = items.length - 1;
+  }
+  items[count].classList.add('active');
+  }
+
+  function plusSlides(n) {
+    showNextItem(items[count] += n);
```

Comment:
here is something not quite right.
lets just reason about this line:
```js
showNextItem(items[count] += n);
```
- `items` in your code is an array of HTML Elements (images)
- `count` in your code is come `index`
- `items[count]` will give us an HTML Element at the index, right?
- then you are trying do this: `items[count] += n` which is the same as `items[count] = items[count] + n`;

so what is the meaning of this?
```js
showNextItem(items[count] = items[count] + n);
```
if (for example) 
- items[count] = <img src=".....">
- n = 5

Reply:
although, the website work, it might become a serious bug in the future ;)